### ISSUE TITLE
Error: internal error: Daemon response did not include the requested real output

### ISSUE THREAD

**Issue by gattacus** [m2049r/xmrwallet#311]
The app is unusable because of this problem. Can't send any amounts. 
also see https://www.reddit.com/r/Monero/comments/8n9t6t/error_internal_error_daemon_response_did_not/
I assume the app needs to integrate the latest core fixes.

**Comment by m2049r** [comment]
yes, this does not seem to happen with the 0.12.2 code - will release in the next couple of days.

**Comment by m2049r** [comment]
fixed in v1.5.5 - pls. verify @gattacus

**Comment by gattacus** [comment]
yes, it works now.
I noticed two other small issues. I couldn't find the close button. I wonder if you removed it on purpose but I don't think it's obvious how to close the wallet. Moreover the top left menu button disappears if you use the task switcher button. At least for me, Samsung S7.
 It would also be nice to delete/wipe the secondary accounts (subaddresses)

**Comment by m2049r** [comment]
thanks for the testing & reporting :)

close button: it's gone on purpose in favour of the hamburger. google suggests not to use back/close as every android device must have a back button. i am thinking about adding a "close wallet" option in the navigation drawer. not sure about that yet though.

hamburger disappearing: an oversight on my part.

delete account: this is something i wanted to do, but it would have to be a workaround as the monero API does not allow for deleting accounts / subaddresses. not sure i want to do that.
